vk-definitions.json

```diff
@@ -192,6 +192,14 @@
                 "GithubOwner": "k14s",
                 "GithubRepo": "ytt",
                 "ReleaseName": "ytt-linux-amd64"
+            },
+	    {
+	        "Cmd": "kubesec",
+                "VersionArg": "version",
```

Comment:
kubesec does not accept the argument `version`. It looks like it accepts `--version`.